src/libs/ReportActionsUtils.js

```diff
@@ -97,13 +97,14 @@ function hasCommentThread(reportAction) {
  * Returns the parentReportAction if the given report is a thread/task.
  *
  * @param {Object} report
+ * @param {Object} [allReportActionsParam]
  * @returns {Object}
  */
-function getParentReportAction(report) {
+function getParentReportAction(report, allReportActionsParam = {}) {
```

Comment:
This default is causing the deploy blocker here: https://github.com/Expensify/App/issues/23202

If `allReportActionsParam` is not passed, it has value `{}` which is truthy, so we never use `allReportActions`. The `allReportActionsParam || allReportActions` doesn't work as intended.

Reply:
thanks you for quickly fixing this!!

Reply:
Thanks for the fix